modules/prebidServerBidAdapter.js

```diff
@@ -96,36 +96,56 @@ function setS2sConfig(options) {
   }
 
   _s2sConfig = options;
-  if (options.syncEndpoint) {
-    queueSync(options.bidders);
-  }
 }
 getConfig('s2sConfig', ({s2sConfig}) => setS2sConfig(s2sConfig));
 
+/**
+ * resets the _synced variable back to false, primiarily used for testing purposes
+*/
+export function resetSyncedStatus() {
+  _synced = false;
+}
+
 /**
  * @param  {Array} bidderCodes list of bidders to request user syncs for.
  */
-function queueSync(bidderCodes) {
+function queueSync(bidderCodes, gdprConsent) {
   if (_synced) {
     return;
   }
   _synced = true;
-  const payload = JSON.stringify({
+
+  let payload = {
```

Comment:
why `const` => `let` here? I see mutation, but no re-assignments... seems like `const` should still work?

Reply:
I'll change it to const to be consistent.